include/pycppad/utils/scope.hpp

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2019-2021 INRIA
```

Comment:
2021 License

Reply:
This is comming from Pinocchio. So I cannot change it.